Configuration/ProcessModifiers/python/perLSsaving_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier sets the perLS saving in process.DQMStore.saveByLumi for nanoDQMIO
+perLSsaving = cms.Modifier()
```

Comment:
How about e.g. `dqmPerLSSaving` to make it clear that the Modifier is related to DQM?

Reply:
Done, thanks for the suggestion!